README.md

```diff
@@ -74,26 +74,48 @@ Environment variables
 
 Real Inputs
 ---------------
-To hook the server up to your real inputs you need to create a configuration file that connects to your input source and applies the relevant parsers / converters in the provider pipeline.
+To hook the server up to your real inputs you need to create a configuration file that connects to your input source and applies the relevant parsers / converters in the provider pipeline.it is
 
-Inputs are configured as an array of pipedProviders, each with an id and an array pipeElements. You need to create a pipedProviders entry for each of your inputs.
+Inputs are configured as an array of pipedProviders, each with an id and an array pipeElements. You need to create a pipedProviders entry for each of your inputs. The pipedProvider starts with a name (name of the interface, e.g. NMEA0183 from AIS could be called "AIS"). The first provider is the source of the data ("providers/serialport" for a serial interface such as USB, "providers/filestream" if the data comes from a file, "providers/tcp" if the source is a TCP port etc.). You will see in the example settings that the second provider is often "providers/liner". This is a provider that splits the input into separate lines and passes one line at a time to the next provider. The final provider or group of providers is where the translation or passing to the server occurs. For NMEA0183, this is "providers/nmea0183-signalk", for N2K (NMEA2000) it is "providers/n2kAnalyzer" and then "providers/n2k-signalk".
+
+There are also special providers such as "providers/log" which logs the output of the previous provider to a file specified in the options, and "providers/throttle" which changes the playback from files to the bytes per second rate set in the options. The "providers/execute" provider lets you pass a command to the server, as set in the options.
 
 Each PipeElement is configured with `options`. Different PipeElements use different configuration parameters, like for serialport you can configure baud rate and for udp connection the port.
 
 A PipeElement may require some options entry that is available already in the configuration file (`nmea0183-signalk` needs the self id). This can be accomplished with [optionMappings property](https://github.com/SignalK/signalk-server-node/blob/5134ce5f48f25b2b270176f39d3e8e530dedb825/settings/volare-udp-settings.json#L34-L43). 
 
 You can also use optionMappings property to optionally override `options` entries with command line parameters. For example you can specify the data file for file playback from the command line as `bin/signalk-server -s settings/volare-file-settings.json --nmeafilename=samples/nais400-merrimac.log` with [this configuration](https://github.com/SignalK/signalk-server-node/blob/5134ce5f48f25b2b270176f39d3e8e530dedb825/settings/volare-file-settings.json#L28-L33).
 
+Look through the examples in the settings folder and copy pipedProviders that suit your setup. You can combine as many as you want.
```

Comment:
Add a link to https://github.com/SignalK/signalk-server-node/blob/master/settings/multiple-sources.json maybe?